app/lib/submission_builder/return_header1040.rb

```diff
@@ -53,7 +53,7 @@ def document
           xml.Filer {
             xml.PrimarySSN intake.primary_ssn
             xml.SpouseSSN intake.spouse_ssn if tax_return.filing_jointly?
-            xml.NameLine1Txt person_name_type(client.legal_name)
+            xml.NameLine1Txt name_line_1_type(intake.primary_first_name, intake.primary_middle_initial, intake.primary_last_name, intake.spouse_first_name, intake.spouse_middle_initial, intake.spouse_last_name)
```

Comment:
lmao, yeah. pretty gross, but i guess its better to pass in the individual strings than the whole object. Although, maybe passing the intake wouldnt be the worst thing in the world... There's a tradeoff because if for some reason you changed the order of one of these parameters, the whole thing would be incorrect, right? so that's something to think about as well.